src/Uno.UI/UI/Xaml/Input/PointerRoutedEventArgs.Android.cs

```diff
@@ -72,6 +78,18 @@ internal PointerRoutedEventArgs(MotionEvent nativeEvent, int pointerIndex, UIEle
 			KeyModifiers = keys;
 			OriginalSource = originalSource;
 
+			// On platforms with managed pointers, we reuse the same PointerRoutedEventArgs instance
+			// as we bubble up the event in the visual tree. On Android, the event bubbling is done
+			// natively and we create a corresponding (new) managed PointerRoutedEventArgs instance
+			// for each element up the tree. This means that parents won't see modifications in the
+			// PointerRoutedEventArgs instance that were done by the children. We have to detect
+			// this and copy the relevant fields ourselves.
+			if (_lastNativeEvent?.nativeEvent == nativeEvent)
+			{
+				GestureEventsAlreadyRaised = _lastNativeEvent.Value.lastArgs.GestureEventsAlreadyRaised;
```

Comment:
It should be the same on iOS, did you validated if you have the same issue? (your fix seems to be on Android only)

Reply:
I can check whether the problem also applies to iOS